tests/airgapped/README.md

```diff
@@ -127,4 +131,10 @@ To test Charmed Kubeflow components in airgapped, follow the instructions in the
 * [Katib](./katib/README.md)
 * [KNative](./knative/README.md)
 * [Pipelines](./pipelines/README.md)
-* [Training Operator](./training/README.md)
\ No newline at end of file
+* [Training Operator](./training/README.md)
+
+You will also need to ensure the airgap setup script is including the file that contains the airgap
+testing images. For a new release, we'll need to create a new dedicated file with the images needed
+for airgap testing the new release.
```

Comment:
I already know what those images are, but I think it might not be clear to others. What if we say here to check the `Test Charmed Kubeflow components in airgapped` section where the tests that use these images are linked?

Reply:
Did a pass in c553807